packages/tools/fetch-tool/src/fluidAnalyzeMessages.ts

```diff
@@ -225,6 +226,7 @@ class DataStructureAnalyzer implements IMessageAnalyzer {
     private readonly dataType = new Map<string, string>();
     private readonly dataTypeStats = new Map<string, [number, number]>();
     private readonly objectStats = new Map<string, [number, number]>();
+    private readonly chunkMap = new Map<string, [string[], number]>();
```

Comment:
i think using an object like this is much more readable
```suggestion
    private readonly chunkMap = new Map<string, {chunks:string[], totalSize:number}>();
```

Reply:
Done